### PR TITLE
#484 Set * box-sizing to border-box

### LINKED ISSUE #484: Add box-sizing border-box universal selector reset to styles.css
```
* {
  box-sizing: border-box;

  &::before,
  &::after {
    box-sizing: border-box;
  }
}
```

### PR DESCRIPTION
**Pull request by eduardseifert**
Fix #484

- Added universal star selector for `box-sizing: border-box` (normalization, reset, streamline)

# Test URLs

- Before: https://main--hlx-test--urfuwo.hlx.page/design/design-system
- After: https://484-box-sizing-border-box--hlx-test--urfuwo.hlx.page/design/design-system (looks the same)

### CODE REVIEW

**Comment by alexcarol** [comment]
The footer looks different:

Main:
image
PR:
https://484-box-sizing-border-box--hlx-test--urfuwo.hlx.page/design/design-system

**Comment by eduardseifert** [comment]
@alexcarol I repaired all the occurences in the footer. Due to the changes to the box-sizing a few changes were made:

# (1) Contact block:

- moved to nested CSS
- changed padding to margin
- A-Z
- formatting

# (2) Footer block

- moved to nested CSS
- changed padding to margin
- A-Z
- formatting
- small UX enhancements
- kept most of the units

Icons are back in place. The main probs resulted in padding (content-box) vs margin (border-box).

File: aemedge/blocks/contact/contact.css [hunk 1]
```diff
@@ -2,80 +2,82 @@
...
-  color: var(--udexColorGrey7);
-  background-color: var(--udexColorNeutralWhite);
-}
-
-.contact-wrapper .icon-sap-logo {
-  height: 32px;
-  width: 65.4px;
-  padding-right: var(--udexSpacer20);
+.contact-wrapper {
+  .logo-lang-picker {
```
Comment (alexcarol) [inline aemedge/blocks/contact/contact.css:6]: Do we have any policies regarding minimum support for css features? This feature seems to be supported in browsers with ~80% of the users (https://caniuse.com/css-nesting), is this enough for this site? /cc @saurabh-khare @mhaack

File: aemedge/templates/article/article.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ function decorate(doc) {
   const main = doc.querySelector('main');
   containerize(main, '.hero');
   const tocFlag = getMetadata('toc');
-  if (tocFlag && tocFlag.content !== 'no' && tocFlag.content !== 'false') {
+  if (tocFlag && tocFlag !== 'no' && tocFlag !== 'false') {
```
[the remark this answers is not in the document]
Comment (eduardseifert) [reply aemedge/templates/article/article.js:10]: This came through the rebase. IDK. @rjwtrmn + @saurabh-khare.

**Comment by eduardseifert** [comment]
@rjwtrmn + @saurabh-khare Also please have a look on this. Due to the changes in box-sizing I had to update contact block and footer block (used the opportunity to rearange the styles).

File: aemedge/templates/article/article.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexcarol) [reply aemedge/templates/article/article.js:10]: I just made a copy of the branch and rebased with main, and I get this diff:
https://github.com/urfuwo/hlx-test/compare/484-box-sizing-border-box-rebase?expand=1
How i did it:
```git fetch origin + git rebase origin/main```

What command did you use to rebase?

**Pull request closed without merging**